### ISSUE TITLE
Import PPD financial data into Polanco

### ISSUE THREAD

**Issue by arborrow** [arborrow/montserrat#181]
Migrate data from Donations to Polanco and create views to show financial contributions (will require creation of permissions, reviewing how financial data is stored in CiviCRM, etc.)

First need to assess various case scenarios (retreat offerings, AGC, bookstore, tips, etc.) and also whether they are one time payments, pledges, or on-going recurring payments.

**Comment by arborrow** [comment]
OK, I discovered that there are 2 tables from ppd that I need to pay attention to: Donations and Donation Payments along with the donation description which serves as the account. Peace - Anthony

**Comment by arborrow** [comment]
Just a quick status update. I have imported most of the data. I will need to meet with Terri and Phil to see how we want to handle payments with no supporting donation record (about 46 of those). There are also a couple of donor records that were duplicated or somewhat confusing. There is a duplicate in Polanco that also needs to be resolved. The donor route can be used to find the appropriate contact id. I have added a section for donations to the contact show view but forgot to tag this issue. I am working on creating a retreat/{id}/payments route to handle processing retreat payments (or at least the bulk of them). The interface will be pretty simple but should cover the majority of the cases. One limitation is a single donation will be recorded so if someone gives a retreat offering and has books that will need to be handled elsewhere as this route is intended for the retreat offering although other options (books, AGC, etc.) will be available.
